app/templates/main/map.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'layouts/base.html' %}
+{% block content %}
+<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
+<script type="text/javascript" src="https://maps.google.com/maps/api/js?sensor=true"></script>
+<script type="text/javascript" src="http://hpneo.github.io/gmaps/gmaps.js"></script>
+    <style type="text/css">
+        #map {
```

Comment:
we shouldn't be really using css in the file like this, it should be put in the sass file at https://github.com/hack4impact/women-veterans-rock/blob/master/app/assets/styles/app.scss. We can talk more about this offline if you're confused.